source/features/pr-jump-to-first-unviewed-file.tsx

```diff
@@ -0,0 +1,34 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function jumpToFirstUnviewed(): void {
+	const firstNotView = select('.js-reviewed-checkbox:not([checked])')!;
+	if (!firstNotView) {
+		// Scroll to page end
+		window.scrollTo(0, document.body.scrollHeight);
+		return;
+	}
+
+	const fileID = 	firstNotView.closest<HTMLDivElement>('[data-anchor]')!.dataset.anchor!;
```

Comment:
This seems to work as well
```suggestion
	const fileID = 	firstNotView.closest('[id]')!.id!;
```